lib/roadie/inliner.rb

```diff
@@ -26,26 +26,30 @@ def initialize(stylesheets, dom)
     #
     # @option options [true, false] :keep_uninlinable_css
     # @option options [:root, :head] :keep_uninlinable_in
+    # @option options [true, false] :use_shared_media_queries
     # @return [nil]
     def inline(options = {})
       keep_uninlinable_css = options.fetch(:keep_uninlinable_css, true)
       keep_uninlinable_in = options.fetch(:keep_uninlinable_in, :head)
+      use_shared_media_queries = options.fetch(:use_shared_media_queries, true)
 
       style_map, extra_blocks = consume_stylesheets
 
       apply_style_map(style_map)
 
       if keep_uninlinable_css
-        add_uninlinable_styles(keep_uninlinable_in, extra_blocks)
+        add_uninlinable_styles(keep_uninlinable_in, extra_blocks, use_shared_media_queries)
       end
 
       nil
     end
 
     protected
+
     attr_reader :stylesheets, :dom
 
     private
+
```

Comment:
I prefer these extra two lines breaks to not be there. (e.g. no empty line break after `protected` and `private`)